GG/GG/Font.h

```diff
@@ -461,35 +508,41 @@ class GG_API Font
     void ProcessTagsBefore(const std::vector<LineData>& line_data, RenderState& render_state,
                            std::size_t begin_line, CPSize begin_char) const;
 
-    /** Returns the maximum dimensions of the string in x and y, and populates
-        \a line_data. */
-    Pt   DetermineLines(const std::string& text, Flags<TextFormat>& format, X box_width,
-                        std::vector<LineData>& line_data) const;
+    /** Populate \p text_elements with TextElements parsed from \p
+        text. \p format onyl determines if tags are ignored. */
+    std::vector<boost::shared_ptr<TextElement> > ComputeTextElements(
+        const std::string& text, const Flags<TextFormat>& format = FORMAT_NONE) const;
 
-    /** Returns the maximum dimensions of the string in x and y, and populates
-        \a line_data and \a text_elements.  Note that \a text_elements must be
-        empty. */
-    Pt   DetermineLines(const std::string& text, Flags<TextFormat>& format, X box_width,
-                        std::vector<LineData>& line_data,
-                        std::vector<boost::shared_ptr<TextElement> >& text_elements) const;
+    /** Fill \p text_elements with the font widths of characters from \p text starting from \p
+        start. */
+    void FillTemplatedText(
+        const std::string& text,
```

Comment:
put first parameter on same line as function name, indent other parameters to match where it starts